great_expectations/expectations/core/schemas/ExpectColumnDistinctValuesToBeInSet.json

```diff
@@ -339,6 +339,7 @@
         },
         "Column": {
             "title": "Column",
+            "description": "--Public API--\nSpecify the column in a condition statement.",
```

Comment:
@NathanFarmer FYI, at one point I got a CI error that said to run `invoke schemas --sync` so I did that and it resulted in these changes to json files.